src/integrations/ActiveCampaign/nativeSdkLoader.js

```diff
@@ -0,0 +1,21 @@
+function loadNativeSdk(actId) {
+  (function (e, t, o, n, p, r, i) {
+    e.visitorGlobalObjectAlias = n;
+    e[e.visitorGlobalObjectAlias] =
+      e[e.visitorGlobalObjectAlias] ||
+      function () {
+        (e[e.visitorGlobalObjectAlias].q = e[e.visitorGlobalObjectAlias].q || []).push(arguments);
+      };
+    e[e.visitorGlobalObjectAlias].l = new Date().getTime();
+    r = t.createElement('script');
+    r.src = o;
+    r.async = true;
```

Comment:
import LOAD_ORIGIN and add a line after 12.
`r.setAttribute('data-loader', LOAD_ORIGIN);`